src/server/index.jsx

```diff
@@ -134,6 +133,18 @@ const injectDefaultCacheHeader = (req, res, next) => {
   next();
 };
 
+const injectResourceHintsHeader = (req, res, next) => {
+  if (req.originalUrl.indexOf('/pidgin') === 0) {
+    const assetOrigins = getAssetOrigins('pidgin');
+    res.set(
+      'Link',
+        assetOrigins.map(
+          domainName =>`<${domainName}>; rel="dns-prefetch", <${domainName}>; rel="preconnect"`,
```

Comment:
would we need tests for this? I struggle to understand the string it builds. Is it `<bbc.com>; rel="dns-prefetch", <bbc.com>; <bbc.co.uk>; rel="dns-prefetch", <bbc.co.uk>;`? is that right? what is it? should the callback function maybe be abstracted out and given a name?